cql3/restrictions/statement_restrictions.hh

```diff
@@ -58,6 +58,10 @@ private:
 
     expr::single_column_restrictions_map _single_column_nonprimary_key_restrictions;
 
+    expr::expression _static_columns_filter = expr::conjunction({});
+    expr::expression _regular_columns_filter = expr::conjunction({});
```

Comment:
Is this default construction used, don't you always assign these two expressions in the statement_restrictions constructor?

Reply:
The expression class doesn't have a default constructor.